### PR TITLE
Gardener

### PR DESCRIPTION
**Pull request by exxaxxxx**
Robot finds in simple way spot to plant hexagonal garden. After that he
waters the trees and is able to build other robots except tank (because
of its size).

### CODE REVIEW

File: src/blueteam/Gardener.java [hunk 1]
```diff
@@ -1,33 +1,163 @@
 package blueteam;
 
-import battlecode.common.Direction;
-import battlecode.common.GameActionException;
-import battlecode.common.RobotController;
-import battlecode.common.RobotType;
+import battlecode.common.*;
```
Comment (hrnr) [inline src/blueteam/Gardener.java]: set number of imports needed for *: 99

File: src/blueteam/Gardener.java [hunk 2]
```diff
@@ -1,33 +1,163 @@
...
+     */
+	boolean findSpot(float radius) {
+	    // get robots in radius
+        RobotInfo[] nearbyRobots = rc.senseNearbyRobots(radius);
+        for (RobotInfo robot : nearbyRobots) {
+            // check for archrons and gardeners around
+            //TODO make cleverer location finding
+            if (robot.getType().equals(RobotType.GARDENER) || robot.getType().equals(RobotType.ARCHON)) {
+                Direction dir;
+                // TODO use constants
```
Comment (hrnr) [inline src/blueteam/Gardener.java]: you can use TeamConstants class to store constatnts if you wish.

**Review by hrnr — CHANGES_REQUESTED** [review]
Formatting is still not perfect, it uses BOTH tabs and spaces which may cause death to all kittens in your neighbourhood.

If you want to extend your git-fu you may want to squash all the fixup commits and remove the merge commit.

**Review by Lukx19 — APPROVED** [review]
formating ok. Constants can be improved in next version

**Review by hrnr — APPROVED** [review]
it looks much better formatted.

**Pull request merged**